src/models/shape_loader.cpp

```diff
@@ -893,9 +911,9 @@ geo::ShapePtr loadShape(const std::string& model_path, tue::config::Reader cfg,
         error << "[ED::MODELS::LOADSHAPE] Error while loading shape with data:" << std::endl << cfg.data() << std::endl;
     }
 
+    // Extra pose is only allowed in ED yaml.
     readPose(cfg, pose);
-
-    if (shape)
+    if (shape && pose != geo::Pose3D::identity())
```

Comment:
This solves issue https://github.com/tue-robotics/ed/issues/95 . However the changes made here are a workaround. I am not saying that you should change it but know that the issue still persists in the case of ED yamls. And when people encounter the problem they might not immediatly spot what is going wrong.

Reply:
True, but to completely get rid of this for the ED yamls too, we would need to break the current 'spec', which I don't want, because we are trying to move away from it.